Analysis/Tasks/PWGHF/HFJpsiToEECandidateSelector.cxx

```diff
@@ -50,24 +38,8 @@ struct HFJpsiToEECandidateSelector {
 
   Configurable<double> d_TPCNClsFindablePIDCut{"d_TPCNClsFindablePIDCut", 70., "Lower bound of TPC findable clusters for good PID"};
   Configurable<double> d_nSigmaTPC{"d_nSigmaTPC", 3., "Nsigma cut on TPC only"};
-
-  /// Gets corresponding pT bin from cut file array
-  /// \param candpT is the pT of the candidate
-  /// \return corresponding bin number of array
-  template <typename T>
-  int getpTBin(T candpT)
-  {
-    double pTBins[npTBins + 1] = {0, 0.5, 1., 2., 3., 4., 5., 7., 10., 15.};
-    if (candpT < pTBins[0] || candpT >= pTBins[npTBins]) {
-      return -1;
-    }
-    for (int i = 0; i < npTBins; i++) {
-      if (candpT < pTBins[i + 1]) {
-        return i;
-      }
-    }
-    return -1;
-  }
+  Configurable<std::vector<double>> ptBins{"ptBins", std::vector<double>{hf_cuts_jpsi_toee::pTBins_v}, "pT bin limits"};
```

Comment:
Please change `ptBins` to `pTBins` to be consistent with the rest of your and other HF code.

Reply:
Ok, done, should I also modify this on D0 part?